### PR TITLE
feat: add trusted icon

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
### Requirements

* Add the trusted icon to the ui-kit

### Description of the Change

The trusted icon has been added to UI Kit

Fixes #MANAGER-5414

### CODE REVIEW

File: packages/base/core/src/svg/icons/concept/trusted_concept.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...rlimit:10;}</style></defs><g transform="translate(-1.01 -1.5)"><path class="a" d="M2627.579-502.489a23.482,23.482,0,0,1-2.6-1.906c-2.632-2.221-3.965-4.342-3.965-6.3v-7.4a2,2,0,0,1,1.724-1.981,2.61,2.61,0,0,1,.356-.023,4.9,4.9,0,0,1,2.039.6,4.909,4.909,0,0,0,1.67.55.761.761,0,0,0,.1-.006,11.75,11.75,0,0,0,1.479-1.7,2,2,0,0,1,1.628-.839,2,2,0,0,1,1.628.839,11.81,11.81,0,0,0,1.479,1.7.761.761,0,0,0,.1.006,4.9,4.9,0,0,0,1.67-.55,4.9,4.9,0,0,1,2.039-.6,2.568,2.568,0,0,1,.355.023,2,2,0,0,1,1.725,1.981v7.4c0,1.963-1.334,4.083-3.965,6.3a23.476,23.476,0,0,1-2.605,1.906,4.6,4.6,0,0,1-2.43.989A4.606,4.606,0,0,1,2627.579-502.489ZM2623.01-518.1v7.4c0,3.137,6.348,7.2,7,7.2s7-4.064,7-7.2v-7.4a.6.6,0,0,0-.081,0c-.722,0-2.186,1.147-3.709,1.147a2.666,2.666,0,0,1-.877-.142,9,9,0,0,1-2.333-2.4,9,9,0,0,1-2.333,2.4,2.666,2.666,0,0,1-.877.142c-1.523,0-2.985-1.147-3.709-1.147A.6.6,0,0,0,2623.01-518.1Zm4.9,9.824-2.219-2.3a1.1,1.1,0,0,1,0-1.517,1.058,1.058,0,0,1,1.495-.041l1.495,1.552,4.124-4.282a1.043,1.043,0,0,1,.736-.328,1.018,1.018,0,0,1,.757.288,1.111,1.111,0,0,1,.04,1.555l-4.892,5.08a1.051,1.051,0,0,1-.573.31l-.165.017A1.11,1.11,0,0,1,2627.911-508.276Z" transform="translate(-2619.5 523.5)"/></g></svg>
```
Comment (antleblanc) [inline packages/base/core/src/svg/icons/concept/trusted_concept.svg:1]: Current existing icons have following details:

- viewBox: `0 0 32 32`
- fill: `#0050D7`

Do you any information about why it's different from here?

Comment (ganeshkumar1989) [reply packages/base/core/src/svg/icons/concept/trusted_concept.svg:1]: I picked this svg from the Mocks. I will check it out with the PU about this. Thanks

Comment (ganeshkumar1989) [reply packages/base/core/src/svg/icons/concept/trusted_concept.svg]: The viewBox has been updated to `0 0 20 20`. I see some icons with this spec. The fill has been set to `#0050D7`.

**Comment by ganeshkumar1989** [comment]
> Hello @ganeshkumar1989, it seems there's a conflict with vendors file

The conflicts are fixed. Thanks :)

**Review by varun257 — DISMISSED** [review]

**Review by radireddy — DISMISSED** [review]

**Comment by cbourgois** [comment]
The icons seems bottom cropped.

**Comment by cbourgois** [comment]
> The icons seems bottom cropped.

My bad seems OK.

**Review by cbourgois — DISMISSED** [review]

**Review by cbourgois — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**